lib/android/app/src/test/java/com/reactnativenavigation/viewcontrollers/ModalStackTest.java

```diff
@@ -103,7 +104,7 @@ public void isEmpty() throws Exception {
     public void onDismiss() throws Exception {
         uut.showModal(viewController, new MockPromise());
         uut.showModal(new SimpleViewController(newActivity(), "otherComponent", new Options()), new MockPromise());
-        uut.dismissAll(new MockPromise());
+        uut.dismissAll();
```

Comment:
What's the reason behind this change?

Reply:
Tests do not wait for animation to finish. And we dismiss dialog in the onAnimationEnd callback. So I'm not sure I know how to fix this test.